living_documentation_generator/generator.py

```diff
@@ -0,0 +1,316 @@
+import logging
+import os
+import shutil
+from datetime import datetime
+
+from living_documentation_generator.action.model.config_repository import ConfigRepository
+from living_documentation_generator.github_integration.github_manager import GithubManager
+from living_documentation_generator.github_integration.model.consolidated_issue import ConsolidatedIssue
+from living_documentation_generator.github_integration.model.issue import Issue
+from living_documentation_generator.github_integration.model.project_issue import ProjectIssue
+from living_documentation_generator.utils import make_issue_key
+
+
+class LivingDocumentationGenerator:
+
+    PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+    ISSUE_PAGE_TEMPLATE_FILE = os.path.join(PROJECT_ROOT, "..", "templates", "issue_detail_page_template.md")
+    INDEX_PAGE_TEMPLATE_FILE = os.path.join(PROJECT_ROOT, "..", "templates", "_index_page_template.md")
+
+    def __init__(self, repositories: list[ConfigRepository],
+                 projects_title_filter: list[str] = None,
+                 project_state_mining_enabled: bool = False,
```

Comment:
default should be True via: `action.yml`.

Reply:
It should be False as the action should provide some basic function by default, and you can enable additional ones. A set up **False** as the default value for now.